### PR TITLE
Output file

### PR DESCRIPTION
**Pull request by TrangEmily**
Add --output-file <fullpath of file> so that the result will be send to a file (only the result goes to the file, other information such as verbose log statements don't).

### CODE REVIEW

File: adr/cli.py [hunk 1]
```diff
@@ -19,13 +19,21 @@
...
+
 def format_request(request, config, remainder, request_type):
 
     if request_type == "query":
         data, url = format_query(request, config, remainder)
     else:
         data = run_recipe(request, remainder, config)
         url = None
+    if config.output_file:
+        print_to_file(data, config.output_file)
```
Comment (ahal) [inline adr/cli.py]: This is supported by the built-in `print` function, no need to create a helper:

```
print("foo", file=open(config.output_file, 'w'))
```

File: adr/cli.py [hunk 2]
```diff
@@ -19,13 +19,21 @@
 log.addHandler(logging.StreamHandler())
 
 
+def print_to_file(data, filepath):
+    f = open(filepath, "x")
```
Comment (ahal) [inline adr/cli.py]: Let's just use 'w' here, otherwise it'll be annoying to run locally.

File: adr/cli.py [hunk 3]
```diff
@@ -96,10 +104,10 @@ def _build_parser_arguments(parser, config):
     # positional arguments
     parser.add_argument('task', nargs='*')
     parser.add_argument(
-        '-h',
-        '--help',
-        action='store_true',
+        '-h', '--help', action='store_true',
         help="Type --help to get help.\nType <recipe> --help to get help with a recipe.")
+    parser.add_argument('-of', '--output-file', type=str,
```
Comment (ahal) [inline adr/cli.py]: I don't think you can use multi-character options with a single `-`. Let's use `-o`

Comment (TrangEmily) [reply adr/cli.py]: The code ran well with "-of", but I changed to "-o" as you suggested. "-o" looks more consistent with other arguments.

File: adr/cli.py (hunk 1 above)
Comment (TrangEmily) [reply adr/cli.py]: Thanks I updated the code 👍

File: adr/cli.py (hunk 2 above)
Comment (TrangEmily) [reply adr/cli.py]: Thanks I updated :)

**Pull request merged**